Project-Descriptions-and-Plans/CVA6/CVA6-plan-approved-2023-2024.md

```diff
@@ -0,0 +1,420 @@
+- The permissive licence scheme that allows the integration in open-source or closed-source projects or the addition of a "secret sauce";
+- The low exposition to export control
+([OpenHW Group Membership Agreement](https://www.openhwgroup.org/membership/openhw-group-membership-agreement-2019-10-16.pdf), section 4.1).
+
+## External dependencies
+
+The project relies on:
+- Related OpenHW projects (joint with other cores): LLVM, FreeRTOS, core-v-verif, CV-X-IF specification
+- Open-source software: GCC, GDB, LLVM, Linux (Yocto, BuildRoot), OpenSBI, UBoot, BBL...
+- Open-source verification: Google Riscv-dv, Spike, RISC-V compatibility tests
+- Open-source hardware: fpnew (ETH Zürich), first verified in CV32E40Pv2
+- Eclipse Foundation, GitHub
+- Digilent Genesys 2 board
+- JADE Design Automation's Register Manager
+- Simulators: Verilator (open-source), Siemens Questa, Synopsys VCS
+- Synthesis: Vivado (Xilinx), Synopsys Design Compiler (ASIC)
+
+CVA6 can be integrated in the OpenPiton framework to build an SMP multi/many-core CPU.
+
+## List of project outputs
+
+### Documentation:
+
+The document structure has been defined at the PL gate:
+- Requirement specification
+    - Identifies features agreed upon
+    - “What” defined as requirements with identifiers
+    - Can be seen as a datasheet
+- Users’ guide
+    - Includes more details than the specification
+    - For CVA6 integrators and users: HW, SW, ASIC, FPGA… viewpoints
+- Design document
+    - Explains the “How”: design choices…
+    - Not prescriptive, written during or after the design
+    - Useful for next projects and certain certification schemes
+- Verification Environment Specification
+    - User manual for the verification environment testbenches, testcases, verification components, etc.
+    - Description of the testbench structure and theory of operation
+- Design Verification Plans
+    - DVplan, Verification Plan, Vplan: same meaning
+    - Feature-by-feature listing of the Device Under Test
+        - and a description of how it will be verified
+        - and how we know when it is verified (coverage).
+    -   Prepared with the [VPTOOL](https://github.com/openhwgroup/core-v-verif/tree/master/tools/vptool) utility
+
+The specification and users' guide are inputs for design verification plans.
+
+### Design 
+
+- CVA6 configurable RTL source code (TRL5 target)
+- Subsystems and FPGA designs, a.k.a. APU, to exercise CVA6 on development boards...
+
+
+### Verification:
+
+- Versatile generic testbench with adaptation layers for CVA6
+    - Subset compliant with _sustainable open-source solution_ expectactions
+- Test sequences
+- Verification results
+     - Including code coverage and functional coverage
+
+### Software:
+
+- Baremetal BSP for FPGA boards featuring CVA6
+- Linux ports (based on UBoot, OpenSBI and Yocto)
+- Toolchains (compiler...)
+
+LLVM and FreeRTOS are products of related OpenHW projects.
+
+
```

Comment:
Again we don't have a separate plan for implementations of CVA6 on Genesys FPGA, can you provide some info here about how prototypes of CVA6 on FPGA are developed and used by the project?

Reply:
Past activity. I do not think it's needed for planning activities.